app/views/shared/_public_dashboard_org_header.html.erb

```
@@ -10,9 +10,9 @@
     <%= @name %>
     <span class="UserInfo-nameBreadcrumb"> /
       <% if @content_type == 'maps' %>
-        <%= link_to 'Maps', @maps_url, class: "DropdownLink js-breadcrumb-dropdown-target" %>
+        <%= link_to 'Maps', '#', class: "DropdownLink js-breadcrumb-dropdown-target" %>
       <% else %>
-        <%= link_to 'Datasets', @datasets_url, class: "DropdownLink js-breadcrumb-dropdown-target" %>
+        <%= link_to 'Datasets', '#', class: "DropdownLink js-breadcrumb-dropdown-target" %>
```

:warning: I think this change will break the pages for users without the feature flag. Won't it?

---

I checked. It's a dummy link. On the original implementation, JS is preventing the default event from happening. 

Because of a refactor on the migration, I have to let the event bubble up, without actually navigating.